addon/-private/utils/element.js

```diff
@@ -61,18 +65,31 @@ export function getScale(element) {
   }
 
   let computedHeightStyleValue = window.getComputedStyle(element).height;
-  let computedHeightInPixels = Number(
-    computedHeightStyleValue.substring(0, computedHeightStyleValue.length - 2)
-  );
+  let computedHeightInPixels = parseComputedStyleHeightToPixels(computedHeightStyleValue);
 
   let offsetHeight = element.offsetHeight;
 
   if (isNaN(computedHeightInPixels)) {
     computedHeightInPixels = offsetHeight;
-  } else if (Math.abs(computedHeightInPixels - offsetHeight) >= 1) {
-    throw new Error(
-      "EmberTable's getScale() utility can only work on elements where height as derived from getComputedStyle is reliable. Usually this means padding is present on the target element."
-    );
+  } else {
+    let [min, max] = [computedHeightInPixels, offsetHeight].sort();
+    let difference = max - min;
```

Comment:
`Math.abs(computedHeightInPixels - offsetHeight)` is simpler

Reply:
Yes but I need `max` below this anyway. 👍